webservices/resources/candidates.py

```diff
@@ -84,7 +98,9 @@ class CandidateSearch(CandidateList):
 
     schema = schemas.CandidateSearchSchema
     page_schema = schemas.CandidateSearchPageSchema
-    query_options = [sa.orm.subqueryload(models.Candidate.principal_committees)]
+    query_options = [
+        sa.orm.subqueryload(models.Candidate.principal_committees),
```

Comment:
We've made a note of this as being something that is still problematic with `nplusone` when changing it to `joinedload` and will address in the future.